### PR TITLE
Run flutter tests with --track-widget-creation as part of the build.

### PR DESCRIPTION
**Pull request by jacob314**
Run tests with --track-widget-creation as part of the build. Previously I ran these manually.

### CODE REVIEW

File: packages/flutter/test/widgets/widget_inspector_test.dart [hunk 1]
```diff
@@ -678,23 +678,23 @@ class TestWidgetInspectorService extends Object with WidgetInspectorService {
       // would make this test fragile.
       expect(lineA + 1, equals(lineB));
       // Column numbers are more stable than line numbers.
-      expect(columnA, equals(21));
+      expect(columnA, equals(15));
```
Comment (jacob314) [inline packages/flutter/test/widgets/widget_inspector_test.dart]: these columns all moved back 6 characters due to removing optional const.
"const ".length == 6

**Comment by jacob314** [comment]
Let me know whether you would like me to only run the widget_inspector_test.dart with this flag or if it is alright to run all the tests with the flag for better coverage in case something goes horribly wrong with the kernel generator.

File: dev/bots/test.dart [hunk 2]
```diff
@@ -153,6 +153,11 @@ Future<Null> _runTests() async {
   await _runSmokeTests();
 
   await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter'));
+  // Only packages/flutter/test/widgets/widget_inspector_test.dart really
```
Comment (goderbauer) [inline dev/bots/test.dart]: Is there a performance impact to this? If no, It should be ok to run all tests with that flag just to get more coverage.

Comment (jacob314) [reply dev/bots/test.dart]: Running the tests with the flag takes only slightly longer than running the tests without the flag.

Comment (goderbauer) [reply dev/bots/test.dart]: OK, let's se what the impact is and if it gets too slow we can always revisit.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**